src/components/FormContainer/Form/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import classNames from "classnames/bind";
+
+const Button = ({ className, isSubmitting }) => {
+    return (
+        <button
+            className={classNames("form__btn", className)}
+            data-btn={isSubmitting ? "wysyłanie..." : "wyślij"}
+            disabled={isSubmitting}
+        >
+            {isSubmitting ? "wysyłanie..." : "wyślij"}
+        </button>
+    );
```

Comment:
nope. It should be more generic. This button with different text suppose to appear multiple times in the code